front/app/components/admin/FormResults/FormResultsQuestion/SentimentQuestion/components/GroupedSentimentScore/GroupedSentimentScore.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
```

Comment:
I'll add unit tests to the util functions on Monday :) It can be reviewed now though!